src/main/java/de/doubleslash/keeptime/view/SettingsController.java

```diff
@@ -234,9 +254,72 @@ private void initialize() {
       resetTaskBarFontButton.setOnAction(ae -> taskBarColor.setValue(Model.ORIGINAL_TASK_BAR_FONT_COLOR));
 
       LOG.debug("aboutButton.setOnAction");
-      aboutButton.setOnAction(ae -> {
-         LOG.info("About clicked");
-         aboutStage.show();
+      initializeAbout();
+   }
+   public void initializeAbout() {

```

Comment:
Settings dialog is quite big now. The space is only needed in the About dialog. Not sure how/if to resolve this

Reply:
Made Settings Dialog smaller by extracting Licenses in an extra Tab